### ISSUE TITLE
Android compilation failed

### ISSUE THREAD

**Issue by EXL** [ptitSeb/gl4es#25]
```
gl4es/src/glx/glx.c:366: error: undefined reference to 'rpi_init'
```

https://github.com/ptitSeb/gl4es/blob/master/src/glx/glx.c#L366

**Comment by lubomyr** [comment]
Try to add in android.mk (where list of source files) line

src/glx/rpi.c \

It will fix compilation problem

https://github.com/pelya/commandergenius/blob/sdl_android/project/jni/gl4es/Android.mk

**Comment by ptitSeb** [comment]
Did I forgot, again, to add some file in Android.mk?

**Comment by ptitSeb** [comment]
Ok, I fixed with it 8dd8dda86e876f5eb929c6101945ecdc03b1401c
But I didn't add that file to Android.mk, as it's not used on Android.

**Comment by EXL** [comment]
Big Thanks!
